### DISCUSSION TITLE
Not able to log into the SOC webpage

### DISCUSSION THREAD

**Discussion by bishop340** [Security-Onion-Solutions/securityonion#3506] [Unsupported Versions]
I have tried updating my current password with so-user update, then enabling the account and even tried creating a new account and still not able to log into the soc.

**Comment by weslambert** [comment]
Are you able to login when adding a new user with `so-user-add $user`?

**Reply by stevencotes** [reply]
No, users are added but they aren't able to log on.
Which log should show the failure?

**Comment by stevencotes** [comment]
Having the same issue.
Account created during setup can access everything and works fine.
Added 2 other accounts using so-user-add, they are created and look ok, but are not getting authenticated on the SOC logon page.
Tried "so-user update" to reset passwords, still not able to get in. Disabled and re-enabled, no change.
Restarted nginx, no change.
Looking for logs now that might show the issue.

**Comment by stevencotes** [comment]
I have determined that I am able to log in using Firefox, but not other browsers.
Going to install our enterprise certs for the UI and check on access.
